.github/workflows/send-email.yml

```diff
@@ -0,0 +1,95 @@
+name: Send communications
+on:
+  push:
+    branches:
+      - production
+    paths:
+      - 'apps/dialtone-documentation/docs/about/whats-new/posts/*.md'
+
+env:
+  HUSKY: 0
+
+jobs:
+  changed-files:
+    name: Get changed files
+    runs-on: ubuntu-latest
+    outputs:
+      added-posts: ${{ steps.changed-blog-posts.outputs.added_files }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Get changed blog posts
+        id: changed-blog-posts
+        uses: tj-actions/changed-files@v42
+        with:
+          files: 'apps/dialtone-documentation/docs/about/whats-new/posts/*.md'
+          json: true
+          escape_json: false
+      - name: List all changed files
+        run: echo '${{ steps.changed-blog-posts.outputs.all_changed_files }}'
+
+  send-email:
+    name: Send email
+    runs-on: ubuntu-latest
+    needs: [changed-files]
+    strategy:
+      matrix:
+        files: ${{ fromJSON(needs.changed-files.outputs.added-posts) }}
+    env:
+      FROM_MAIL: julio.ortega@dialpad.com
+      TO_MAIL: dialtone-users@dialpad.com
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Get link from file name
+        id: get-post-url
+        env:
+          MARKDOWN_FILE_PATH: ${{ matrix.files }}
+        run: |
+          PREFIX="https:\/\/dialtone.dialpad.com"
+          echo "POST_URL=$(echo $MARKDOWN_FILE_PATH | sed -e "s/apps\/dialtone-documentation\/docs/$PREFIX/" -e 's/md/html/')" >> $GITHUB_OUTPUT
+
+      # This will do the following:
+      # - Remove frontmatter.
+      # - Remove script setup.
+      # - Remove blog post component.
+      # - Remove empty lines at the beginning of the file.
+      # - Ellipsis the article to the first 5 lines.
+      # - Append the URL of the article at the end of the article.
+      - name: Process Markdown file
+        env:
+          MARKDOWN_FILE_PATH: ${{ matrix.files }}
+          POST_URL: ${{ steps.get-post-url.outputs.POST_URL }}
+        run: |
+          sed -i '/^---$/,/^---$/d' $MARKDOWN_FILE_PATH
+          sed -i '/^<script setup>$/,/^<\/script>$/d' $MARKDOWN_FILE_PATH
+          sed -i '/^<\/*BlogPost.*$/d' $MARKDOWN_FILE_PATH
+          sed -i '/./,$!d' $MARKDOWN_FILE_PATH
+          sed -i 5q $MARKDOWN_FILE_PATH
+          echo -e "... Read the full article [here]($POST_URL)" >> $MARKDOWN_FILE_PATH
+
+      - name: Send email
+        uses: dawidd6/action-send-mail@v3
+        with:
+          server_address: smtp.gmail.com
+          server_port: 465
+          secure: true
+          username: ${{ env.FROM_MAIL }}
+          password: ${{ secrets.MAIL_PASSWORD }}
```

Comment:
this is not a good solution. It is tying itself to your personal username and pw.
I would suggest using AWS SES for this. No idea if there is already an action for that or not.

Reply:
Lets consider https://github.com/marketplace/actions/action-to-send-email-via-aws-ses-without-using-smtp-credentials

We should have the GHA be reviewed by security team to ensure its not doing nefarious actions with our SES account or content we intend to send.

Reply:
Yeah, that sounds like a better solution, I asked IT several days ago and they haven't replied to my message, so we went with this solution